ddtrace/contrib/rq/__init__.py

```diff
@@ -0,0 +1,244 @@
+"""
+The RQ integration will trace your jobs.
+
+Usage
+~~~~~
+
+The rq integration is enabled automatically when using
+:ref:`ddtrace-run<ddtracerun>` or :ref:`patch_all()<patch_all>`.
+
+Or use :ref:`patch()<patch>` to manually enable the integration::
+
+    from ddtrace import patch
+    patch(rq=True)
+
+
+Worker Usage
+~~~~~~~~~~~~
+
+``ddtrace-run`` can be used to easily trace your workers::
+
+    DD_RQ_WORKER_SERVICE=myworker ddtrace-run rq worker
+
+
+See https://github.com/DataDog/trace-examples/tree/kyle-verhoog/rq/python/rq
```

Comment:
Have to change this URL out and merge the PR in trace-examples

Reply:
is this still a TODO?

Reply:
We don't often refer to trace-examples in our documentation so I would be fine with removing here.